query-engine/query-engine-node-api/src/engine.rs

```diff
@@ -293,19 +287,14 @@ impl QueryEngine {
 
     /// If connected, attempts to start a transaction in the core and returns its ID.
     #[napi]
-    pub async fn start_transaction(&self, input: String, trace: HashMap<String, String>) -> napi::Result<String> {
+    pub async fn start_transaction(&self, input: String, _trace: HashMap<String, String>) -> napi::Result<String> {
```

Comment:
Any reason to keep `trace`?

Reply:
I am actually not sure if removing it will break the JS surface, and I don't have any JS engineer to ask :|

Reply:
Okay, maybe hold off until you can, but if not it's also not a big deal, we can remove it down the line.

Reply:
Actually I think it can be removed, I will check that.

Reply:
btw, this passes integration tests in Prisma, so I assume is safe.